lib/src/main/java/com/futurewei/alcor/common/db/repo/ICacheRepository.java

```diff
@@ -28,5 +29,7 @@
 
     void addItem(T newItem) throws CacheException;
 
+    //void addItems(List<T> items) throws CacheException;
```

Comment:
Btw, so Ignite doesn't support a bulk insertion?

Reply:
Ignite support a bulk insertion,  but if I add this line, all the classes that implement ICacheRepository fail to compile.

Reply:
In that case, we could in a project-wide search for ICacheRepository. Once we turn on the line, we add one missing method for each occurrence and mark them "Not Implemented".

This is not a urgent matter, but we will need bulk insertion eventually.